clang/test/Sema/offsetof-ucrt.c

```diff
@@ -0,0 +1,17 @@
+// RUN: %clang_cc1 %s -triple i686-pc-win32 -fsyntax-only -verify -fms-compatibility
```

Comment:
We should have a RUN line which is not `-fms-compatibility` to show we don't define it in that case.

Reply:
Added. I decided to add the positive and negative tests together. Luckily, UCRT's `<stddef.h>` selects `__builtin_offsetof` when `_MSC_VER` isn't defined.